### ISSUE TITLE
Segmentation Fault in 2.0.0-beta

### ISSUE THREAD

**Issue by ahacking** [sass/node-sass#661]
Using current master, node-sass seg-faults on inclusion of Thoughtbots Neat grid v1.5 (old libSass 3.2 based).  The previous 0.9 - 1.x releases of node-sass do not segfault.

Files to reproduce available at this [Gist](https://gist.github.com/ahacking/f1daa1ce04608e8a6f66).

**Comment by am11** [comment]
Hi @ahacking, thanks for reporting this. But unfortunately, this is a very vague description and is almost like finding needle in haystack.

Please narrow it down to minimum lines of Sass code and pure node-sass call to reproduce this issue.

**Comment by ahacking** [comment]
Ok I've narrowed it down to the following SCSS that causes a segfault:

```scss
@if length($foo) % 2 == 0 {
}
```

The actual line in Neat which triggers the segfault is [here](https://github.com/thoughtbot/neat/blob/v1.5.1/app/assets/stylesheets/functions/_new-breakpoint.scss#L7)

This led me to look at the libsass source, it seems they have fixed a segfault [here](https://github.com/sass/libsass/commit/f2364442d26fa615b29975d97600f291446eab32) for size handling only a few days ago.

I updated the src/libsass submodule in my local repo and rebuilt to see if it would fix the issue but still getting a segfault.

Anyway it appears that it is specific to libsass, and not node-sass so closing.

So seems node-sass just needs to update to the latest libsass.
